pkg/operator/ceph/cluster/controller.go

```diff
@@ -694,3 +752,49 @@ func ClusterOwnerRef(namespace, clusterID string) metav1.OwnerReference {
 		BlockOwnerDeletion: &blockOwner,
 	}
 }
+
+//GetAllStorageNodes returns an array of v1.Node objs this cluster has osds on
```

Comment:
```suggestion
// GetAllStorageNodes returns an array of v1.Node objs this cluster has osds on
```

Reply:
Updating all comments to start with a space.